internal/pkg/cli/app_delete.go

```diff
@@ -273,6 +238,14 @@ func (o *deleteAppOpts) deleteSSMParam() error {
 }
 
 func (o *deleteAppOpts) deleteWorkspaceFile() error {
+	// Return if manifest does not exist.
+	exist, err := o.workspaceService.AppExist(o.AppName)
+	if err != nil {
+		return fmt.Errorf("check if app file %s exists: %w", o.AppName, err)
+	}
+	if !exist {
+		return nil
+	}
```

Comment:
Should we instead update the `DeleteFile` method to handle this scenario? If the file you ask to delete doesn't exist then return a `nil` error?

Reply:
I was debating on this. But eventually the reason why I decided to put this out is because I think the workspace package should only be responsible for basic CRUD operation, if we do more than that, it might harm the reusability.

Reply:
An existence check seems pretty generic to me. If the file doesn't exist you can't delete it. I'd agree with you if it was something potentially inconsistent.

Reply:
It would be simpler but it might not be what we want. So what I envision about `delete` is that every time a user run `delete` no matter what the current status they are, they can always completely remove the application. So as for this question, if a user they accidentally delete the local file themselves and they run "app delete" to delete all the other resources, it does not make sense for them to get error saying "error because the file does not exist", since the original purpose for the `delete` command includes deleting the file. They just do things in advance.

However, there's a pitfall for `delete` is we rely too much on SSM store. If users delete parameters from there, there's no way for them to track their apps and delete the rest of those resources.